Havit.Blazor.Components.Web.Bootstrap/Forms/Internal/HxInputDateRangeInternal.razor.cs

```diff
@@ -12,6 +12,8 @@ public partial class HxInputDateRangeInternal : InputBase<DateTimeRange>, IAsync
 
 		[Parameter] public string InputCssClass { get; set; }
 
+		[Parameter] public string InvalidCssClass { get; set; }
+
```

Comment:
We don't need a parameter for this. Just use `HxInputBase<object>.InvalidCssClass` where needed.